app/controllers/NotificationController.scala

```diff
@@ -29,14 +29,16 @@ import play.api.libs.json._
 import play.api.mvc.Action
```

Comment:
just wondering, if this is being replaced by AuthAction. it might be time to remove this import